### DISCUSSION TITLE
Airbank: Full-Stack Developer Positions

### DISCUSSION THREAD

**Discussion by patrickcneuhaus** [prisma/prisma#8247] [Q&A]
Hey guys!

I'm Patrick, CTO @ [Airbank](https://joinairbank.com). We're a Berlin-based fintech building leveraging Open Banking to build a cash management platform for SMEs. We're currently a team of 12 amazing people and Techcrunch just [announced our seed round](https://techcrunch.com/2021/06/22/airbank-centralizes-all-your-business-bank-accounts-and-financial-data/).

We're looking for full-stack developers who are motivated by building something big and joining a rising company in such an early stage. The setup is fully remote, and we put a lot of value on flexibility and work-life balance.

Our current tech stack is composed primarily of:
- Prisma/Postgres
- Vue
- TypeScript
- GraphQL
- GCP

If you're interested, just shoot me an email at patrick@joinairbank.com or comment on this discussion!

**Comment by ryands17** [comment]
@patrick-emmanuel 👋 

You can also post this on our Slack group in the `#job-listings` channel :)
https://slack.prisma.io/

**Reply by patrickcneuhaus** [reply]
Thanks for the tip!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
